hls4ml/templates/vivado/nnet_utils/nnet_activation.h

```diff
@@ -226,36 +223,38 @@ void softmax(data_T data[CONFIG_T::n_in], res_T res[CONFIG_T::n_in]){
     bool initialized = false;
     typename CONFIG_T::exp_table_t exp_table[CONFIG_T::table_size];
     typename CONFIG_T::inv_table_t invert_table[CONFIG_T::table_size];
+	#pragma HLS resource variable=exp_table core=ROM_nP_BRAM
```

Comment:
Are you happy with this being hardcoded since it is not configurable?

Also, spaces instead of tabs.

Reply:
Whoooooops that pragma was not needed anymore.